src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1822,6 +1822,9 @@ def create_app_service_plan(cmd, resource_group_name, name, is_linux, hyper_v, p
                               reserved=(is_linux or None), hyper_v=(hyper_v or None), name=name,
                               per_site_scaling=per_site_scaling, hosting_environment_profile=ase_def)
 
+    if sku.upper() in ['WS1', 'WS2', 'WS3']:
```

Comment:
@StrawnSC given CLI commands are supposed to idempotent, what is the experience if an existing webapp/functionapp tries to use the LogicApp specific SKUS? Would be good to add some exceptional handling existing non logic apps Serverfarms to try to change to the new SKUs added. Please check with LA team if there is some property on ASPs to identify as LogicApp specific?

Reply:
Yikes it succeeded in updating the SKU to WS1 when I tried the command on a normal ASP that already exists. I'll ask about how to distinguish logic app ASPs
Thanks for pointing that out, I hadn't considered a user trying to update an ASP with the command

Reply:
Ouch - please file a bugs on Management loop (API) to handle not updating in this case?

Reply:
I put a bug report on the control plane ADO and added the client-side validation in the CLI 
According to @archidda the best way to tell if an app service plan is a logic app plan is to just check if `plan.sku.tier == 'WorkflowStandard'`